src/pages/result/index.tsx

```diff
@@ -1,106 +1,65 @@
 import styled from '@emotion/styled'
-import { useMedia } from '@offer-ui/react'
 import type { NextPage } from 'next'
-import { useEffect, useState } from 'react'
-import { useGetInfinitePostsQuery } from '@apis'
+import { useState } from 'react'
+import type {
+  Filters,
+  HandleChangeFilter
+} from '@components/result/FilterSelect/types'
+import { useGetCategoriesQuery, useGetInfinitePostsQuery } from '@apis'
 import {
   ResultHeader,
-  ProductList,
   CategorySlideFilter,
-  FilterSelect
+  FilterSelect,
+  ProductList
 } from '@components'
-import { useCategoryFilterList, useSelectBoxFilter } from '@hooks'
 
 const DEFAULT_POST_PAGE_NUMBER = 8
 
-const Result: NextPage = () => {
-  const { desktop } = useMedia()
-  const [isDesktop, setIsDesktop] = useState(false)
-
-  useEffect(() => {
-    if (desktop) {
-      setIsDesktop(true)
-    } else {
-      setIsDesktop(false)
+const ResultPage: NextPage = () => {
+  const getCategoriesQuery = useGetCategoriesQuery()
+  const categories =
+    getCategoriesQuery.data?.map(({ code, name }) => ({ code, name })) || []
+  const [filters, setFilters] = useState<Filters>({
```

Comment:
```suggestion
  const [filters, setFilters] = useState<Filters>({
```
searchOptions로 하는 건 어떠신가요!?

Reply:
[변경 완료](https://github.com/price-offer/offer-fe/pull/238/commits/47819e40005624da1b2f6c299f1a0ed1875a98a1#diff-33ad00554e4d8335fcb7f35a52e38907ad21791c810c9615fe473c46ff271abcR22) !